docs/configuration.md

```diff
@@ -0,0 +1,541 @@
+---
+custom_edit_url: https://github.com/stryker-mutator/stryker-net/edit/master/docs/configuration.md
+---
+
+## Basics
+
+You run stryker from the test project directory.
+
+On some dotnet core projects stryker can run without specifying any custom configuration. Simply run `dotnet stryker` to start testing.  
+On dotnet framework projects the solution path argument is always required. Run at least `dotnet stryker --solution <solution-path>` or specify the solution file path in the config file to start testing. See [solution](#solution-<path>).
```

Comment:
None of the links to option headings are working for me.

Reply:
Mhm I see what you mean, they're not working on github for me either. Perhaps it's something to specifically with the github pull request diff window? Like I said they work fine in vscode. More importantly the links need to work on the website, I'll look up what link syntax docusaurus supports.